docs/serving/autoscaling/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Autoscaling"
+linkTitle: "Autoscaling"
+weight: 20
+type: "docs"
+aliases:
+    - ./docs/serving/configuring-autoscaling.md
```

Comment:
I don't think you want this to be `../configuring-autoscaling` or `/docs/serving/configuring-autoscaling`; this would make an alias of `/docs/serving/autoscaling/docs/serving/configuring-autoscaling.md`.